conpix.cpp

```diff
@@ -23,7 +23,7 @@ using namespace std;
 
 bool fexists(const char *filename) {
   ifstream ifile(filename);
-  return ifile;
+  return ifile.is_open();
```

Comment:
this code was working last i used it. basically it returns true if ifile is not NULL. did you have any issue with it ?